Meta: Link to changelog and Linus' Serenity VNC

BenWiederhake opened the pull request.

supercomputer7 commented on Meta/Websites/serenityos.org/index.html:
```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
+    <li><a href="https://changelog.serenity/">Lubrsi's commit overview, grouped by month and category</a></li>
```
I think you meant `https://changelog.serenityos.org`?

BenWiederhake replied:
Derp >_<

linusg commented on Meta/Websites/serenityos.org/index.html:
```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/2nd/">Happy 2nd birthday! SerenityOS: The second year</a></li>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
```
Please don't. I don't want to give any guarantees about uptime, and too many people using it makes the single instance unusable really quickly (even from a shared keyboard/mouse perspective). It's ok when people find it from Discord or the original announcement though, but I don't want it linked :)

BenWiederhake replied:
Ah, sorry, then I misunderstood you. Sure!

BenWiederhake commented:
Changes since initial push:
- Removed the reverence to the VNC instance
- Fixed the URL (How the hell did I manage to get that wrong in the first place!)

IdanHo commented:
super nitpick: the commit mentions sub-projects in the plural, while only one is added

BenWiederhake commented:
Changes since last push:
- Changed the commit title for super-hardcore nitpick

IdanHo reviewed (approved):
Thanks!
